components/sections/organizations/Organizations.tsx

```diff
@@ -0,0 +1,50 @@
+// React/Relay
+import React, { FC } from "react";
+import { createFragmentContainer, graphql } from "react-relay";
+import singleFragmentComponent from "../../../lib/singleFragmentComponent";
+
+// Component
+import { Props } from "./Organizations.types";
+import styles from "./Organizations.scss";
+
+// External
+import Organization from "./Organization";
+
+export const Organizations: FC<Props> = (props) => (
+  <section className={styles.section}>
+    <div className={styles.titleWrapper}>
+      <h1 className={styles.title}>{props.section_name}</h1>
+    </div>
+    <div className={styles.organizations}>
+      {props.organizations.map((item, index) => (
+        <div className={styles.organizationWrapper} key={index}>
+          <Organization data={item.organization} />
+          {index + 1 < props.organizations.length && (
```

Comment:
you can set this in css as a ::after and set 
```
&:last-child {
   display: none;
}
```

Reply:
Can you give me an example of this?
Are you saying that we should render the line using a content property in the CSS file?

Reply:
Yes i think you can do it with an ::after selector like

```
.organizationWrapper {
 position: relative;
...other styles
   &::after {
    width: line-width;
    height: line-border
    content: '';
    position: absolute;
    bottom: 0;
  }

   &:last-child::after {
      content: none;
  }
}
```

I think `content: none` will work but if not `display: none`